client/src/reducers/genesetsUI.ts

```diff
@@ -1,15 +1,20 @@
+import { AnyAction } from "redux";
+
+export interface GeneSetsUIState {
+  createGenesetModeActive: boolean;
+  isEditingGenesetName: string | false;
+  isAddingGenesToGeneset: string | false;
```

Comment:
This seems overloaded, might be smoke showing that we need to refactor something. Could you look into it really quick and create a ticket if necessary.

Reply:
I don't think they're overloaded - They're used for different things